composer/workflows/cloud-build.yaml

```diff
@@ -0,0 +1,5 @@
+# Job for keeping the test Python packages in the Test Cloud Composer Environment up to date
+steps:
+  - name: 'gcr.io/cloud-builders/gcloud'
+    args: ['composer', 'environments', 'update', 'composer-test', '--update-pypi-packages-from-file', 'requirements-composer.txt', '--location', 'us-central1']
+timeout: 3600s #set timeout to 1h. It's unusual for installing Python packages in a composer environment to take 1h to update. 
```

Comment:
Does the timeout need to be this high?

Reply:
We could lower it to 30 or 45 min - the Composer operation finishes no matter what the timeout is, but Cloud Build's default timeout is 10 min, and [it will show an error-y message](https://screenshot.googleplex.com/7VEYdcZntY2VEdq) saying "the build timed out" unless we increase the timeout. The Composer operation takes between 20-30 min usually - the 60 min timeout is me being generous. The 60 min does not mean that it will take 60 min every time

Reply:
would you prefer something that's closer to the actual expected time rather than a really generous one?

Reply:
That makes sense to me (I also prefer the more generous timeout). Just wanted to double-check. :)